### ISSUE TITLE
Seeing 60 FPS while app is running on 144Hz

### ISSUE THREAD

**Issue by jenshor** [gskinnerTeam/flutter-stats-fl#8]
I'm running a MacOS app on a 144Hz screen in Profile Mode. While my profiler shows 144 FPS average, I only see 60 FPS within `StatsFl`.

Once the framerate drops to below 60 (due to heavy and unoptimised animations), i'm seeing values similar to the ones provided by the profiler.

<img width="755" alt="image" src="https://user-images.githubusercontent.com/16862002/175096996-ee0802af-4834-4258-8c89-8572ae89237f.png">

**Comment by esDotDev** [comment]
Did you try setting `maxFps` to 144?

**Comment by jenshor** [comment]
Yes. I set `maxFps` to 144. I also debugged into the `StatsFl` widget and 144 was applied as well.

**Comment by jenshor** [comment]
However, while debugging, I wasn't able to pinpoint the problem.

**Comment by jenshor** [comment]
I just disconnected my external 144Hz display and used the Build-In ProMotion (variable refresh rate up to 120 Hz) display. While I cant force the display to run at 120 hz constantly, I was able to force 60 and 48 Hz. In both cases, the counter just showed stable 60 Hz. It might be that Flutter does not detect that my system is 120/144 Hz capable and just defaults to 60 Hz. 

The profiler might be able to detect the system's capability, but i'm assuming that frame rate is just calculated based on frame time and then capped to the max capable framerate of the system. 

Any thoughts/experience with this? 
Since my animation runs at 60 FPS I really dont need to optimize further, but I think the topic is interesting anyways 😀
